ajax/libs/ckeditor/package.json

```diff
@@ -17,5 +17,20 @@
     "deprecated_LGPL-2.1+",
     "MPL-1.1"
   ],
-  "author": "CKSource (http://cksource.com/)"
+  "author": "CKSource (http://cksource.com/)",
+  "npmName": "ckeditor",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(adapters|skins|plugins|lang)/**/*.+(js|css|png|gif|map|jpg)"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "+(build-config|ckeditor|config|contents|styles).+(js|css|map)"
+      ]
+    }
+  ]
```

Comment:
Make two filemap but the same base path obiviously makes no sense, why not merge them like this?
```js
"files": [
  "+(adapters|skins|plugins|lang)/**/*.+(js|css|png|gif|map|jpg)",
  "+(build-config|ckeditor|config|contents|styles).+(js|css|map)"
]
```